.gitignore

```diff
@@ -45,7 +45,7 @@ app/cdk.out/
 app/lib/medical-transcription-analysis-client-stack.js
 app/lib/medical-transcription-analysis-client-stack.js-e
 app/lib/medical-transcription-analysis-stack.js
-
-
-
+app/lambda/__pycache__/datastore.cpython-37.pyc
```

Comment:
you might need to delete the __pycache__ files that were already pushed in previous commits.